core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -915,9 +915,9 @@
               </xs:annotation>
             </xs:element>
           </xs:sequence>
-          <xs:attribute name="index" type="tns:indexing" default="${Indexing.index}">
+          <xs:attribute name="enabled" type="xs:boolean" default="false">
```

Comment:
Not using the magical defaults :) Afraid of it?

Reply:
This will always be false and it takes just a millisecond to write "false" there  :)

Reply:
Somebody from the future might change the java side and then the xsd will be out of sync :)

Reply:
What does ```${Indexing.index}``` mean? Is it IndexingConfiguration.index? If so, the default was ```null```

Reply:
Setting ```false``` here is safer

Reply:
Right, let's keep ```false``` without the magic. Nobody , even from the future, should want to change this default because indexing-by-default is not really a good idea.

Reply:
I'm more and more inclined to remove those ${majik} things and make the maven plugin just validate that both java and xsd have same default value or fail the build otherwise. Would finally make the xsd valid in intellij.